cobbler/items/item.py

```diff
@@ -154,6 +154,28 @@ def __eq__(self, other):
             return self._uid == other.uid
         return False
 
+    def __setattr__(self, name, value):
+        """
+        Intercepting an attempt to assign a value to an attribute.
+
+        :name: The attribute name.
+        :value: The attribute value.
+        """
+        if (
```

Comment:
I'd appreciate it if we could document why these conditions are used here. It'll help when we look at this function in a few months from now.

Reply:
Added comments to `if`.